src/constants/color.ts

```diff
@@ -0,0 +1,26 @@
+export const COLOR = {
+  PRIMARY: '#f76c5e',
+  WHITE: {
+    _100: '#ffffff',
+    _200: '#fffef9',
+    _300: '#fff6e9',
+  },
+  ORANGE: {
+    _200: '#feceab',
+    _300: '#f68e5f',
+  },
+  GREEN: {
+    _300: '#58b368',
+  },
+  BLUE: {
+    _300: '#586ba4',
+  },
+  PURPLE: {
+    _300: '#b371d0',
+  },
+  GRAY: {
+    _100: '#cccccc',
+    _200: '#2a363b84',
+    _300: '#2a363b',
+  },
```

Comment:
호출할때 .을 두번 써야해 불편해 보이기도했고 css도 나열하는감이 있어 통일성을 위해 아래와 같이가 좋을 것 같다고 생각했으나 막상보니 민우님이 하신게 더 보기는 좋은것 같기도하고 잘 모르겠습니다 ㅎㅎ,,
```suggestion
  PRIMARY: '#f76c5e',
  WHITE_100: '#ffffff',
  WHITE_200: '#fffef9',
  WHITE_300: '#fff6e9',
  ORANGE_200: '#feceab',
  ORANGE_300: '#f68e5f',
  GREEN_300: '#58b368',
  BLUE_300: '#586ba4',
  PURPLE_300: '#b371d0',
  GRAY_100: '#cccccc',
  GRAY_200: '#2a363b84',
  GRAY_300: '#2a363b',
  },
```

Reply:
저도 밑에 쪽이 좋으려나 쓰..